src/policy/policy.cpp

```diff
@@ -101,38 +111,32 @@ bool IsStandardTx(const CTransaction& tx, bool permit_bare_multisig, const CFeeR
         // 20-of-20 CHECKMULTISIG scriptPubKey, though such a scriptPubKey
         // is not considered standard.
         if (txin.scriptSig.size() > MAX_STANDARD_SCRIPTSIG_SIZE) {
-            reason = "scriptsig-size";
-            return false;
+            MAYBE_REJECT(reason, "scriptsig-size", ignore_rejects);
         }
         if (!txin.scriptSig.IsPushOnly()) {
-            reason = "scriptsig-not-pushonly";
-            return false;
+            MAYBE_REJECT(reason, "scriptsig-not-pushonly", ignore_rejects);
         }
     }
 
     unsigned int nDataOut = 0;
     TxoutType whichType;
     for (const CTxOut& txout : tx.vout) {
         if (!::IsStandard(txout.scriptPubKey, whichType)) {
-            reason = "scriptpubkey";
-            return false;
+            MAYBE_REJECT(reason, "scriptpubkey", ignore_rejects);
         }
 
         if (whichType == TxoutType::NULL_DATA)
             nDataOut++;
         else if ((whichType == TxoutType::MULTISIG) && (!permit_bare_multisig)) {
-            reason = "bare-multisig";
-            return false;
+            MAYBE_REJECT(reason, "bare-multisig", ignore_rejects);
```

Comment:
[The `else` below needs to go too, or we accidentally also ignore dust multisigs](https://github.com/luke-jr/bitcoin/commit/e786e726d4a0b8e68aa87bd38dfac4aeecc76be6#diff-ea6d307faa4ec9dfa5abcf6858bc19603079f2b8e110e1d62da4df98f4bdb9c0R174)